edge/pkg/servicebus/servicebus.go

```diff
@@ -10,10 +10,10 @@ import (
 	"sync/atomic"
 	"time"
 
-	"github.com/astaxie/beego/orm"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/klog/v2"
 
+	"github.com/astaxie/beego/orm"
```

Comment:
"github.com/astaxie/beego/orm"  should be placed at the second part which is for third-party library.  The third part is for kubeedge itself.